src/NHibernate.Test/NHSpecificTest/GH1180/FixtureByCode.cs

```diff
@@ -90,9 +90,9 @@ public void DecimalTypes()
 			using (var session = OpenSession())
 			using (var transaction = session.BeginTransaction())
 			{
-				session.Save(new Entity {Amount = 3.141m});
-				session.Save(new Entity {Amount = 42.131m});
-				session.Save(new Entity {Amount = 17.991m});
+				session.Save(new Entity {Amount = 3.14m});
+				session.Save(new Entity {Amount = 42.13m});
+				session.Save(new Entity {Amount = 17.99m});
```

Comment:
I have not checked if it makes sense within this test to check if the expected scale is respected, but "fixing" SQLite this way cause the test to no more check this point.

If it was making sense, it is not what should be done. In such case, the test should instead be adapted by checking if the dialect has proper support of decimal.

Reply:
It's not exactly related. For this test only constant projection scale check make sense for `Projections.Constant(20.123m)`. But still I restored this check too.